nservicebus/dependency-injection/structuremap.md

```diff
@@ -12,19 +12,7 @@ related:
 ---
 
 
-NServiceBus can be configured to use [StructureMap](https://structuremap.github.io/) for dependency injection.
-
-
-### Default usage
-
-snippet: StructureMap
```

Comment:
Decided to remove snippets for config since they just repeat official SM doco and we still have the sample that shows it.